server/cluster/manager.go

```diff
@@ -290,13 +290,37 @@ func (m *managerImpl) GetTablesByShardIDs(clusterName, _ string, shardIDs []stor
 	return shardTables, nil
 }
 
+// DropTable is only used for the HTTP interface.
+// It only deletes the table data in ETCD and does not initiate a table deletion request to CeresDB.
 func (m *managerImpl) DropTable(ctx context.Context, clusterName, schemaName, tableName string) error {
 	cluster, err := m.getCluster(clusterName)
 	if err != nil {
 		return errors.WithMessage(err, "get cluster")
 	}
 
-	_, err = cluster.metadata.DropTable(ctx, schemaName, tableName)
+	table, ok, err := cluster.GetMetadata().GetTable(schemaName, tableName)
+	if !ok {
+		return metadata.ErrTableNotFound
+	}
+	if err != nil {
+		return errors.WithMessage(err, "get table")
+	}
+
+	getShardNodeResult, err := cluster.GetMetadata().GetShardNodeByTableIDs([]storage.TableID{table.ID})
+	if err != nil {
+		return errors.WithMessage(err, "get shard node by tableID")
+	}
+	if len(getShardNodeResult.ShardNodes[table.ID]) != 1 || len(getShardNodeResult.Version) != 1 {
+		return metadata.ErrShardNotFound
+	}
+
+	shardID := getShardNodeResult.ShardNodes[table.ID][0].ID
+	version := getShardNodeResult.Version[shardID]
+
+	_, err = cluster.metadata.DropTable(ctx, schemaName, tableName, metadata.ShardVersionUpdate{
```

Comment:
Is there any different between `cluster.GetMetadata()` and `cluster.metadata`?

Reply:
`cluster.GetMetadata()` is used to get `metadata` in other package, replace all `cluster.GetMetadata()` in this package to `cluster.metadata`